lib/components/narrative/access-leg.js

```diff
@@ -12,48 +12,58 @@ export default class AccessLeg extends Component {
     setActiveLeg: PropTypes.func,
     setActiveStep: PropTypes.func
   }
+
   _onLegClick (e, leg, index) {
     if (this.props.active) {
       this.props.setActiveLeg(null)
     } else {
       this.props.setActiveLeg(index, leg)
     }
   }
+
   _onStepClick (e, step, index) {
     if (index === this.props.activeStep) {
       this.props.setActiveStep(null)
     } else {
       this.props.setActiveStep(index, step)
     }
   }
+
   render () {
     const { active, activeStep, index, leg } = this.props
+    const legClickHandler = (e) => this._onLegClick(e, leg, index)
     return (
       <div
         key={index}
         className={`leg${active ? ' active' : ''}`}>
-        <div
+        <a
```

Comment:
`a`s don't work here (click doesn't trigger div expansion).  Also, previously a click anywhere in the dev triggered expansion, now you have to click exactly on the link (except that it doesn't work at the moment)
![image](https://cloud.githubusercontent.com/assets/2370911/23811947/2fe4f89a-05a6-11e7-9821-54e05673dfb1.png)